Intuit.TSheets/Model/Jobcode.cs

```diff
@@ -193,5 +193,12 @@ public Jobcode(string name)
         /// </summary>
         [JsonProperty("project_id")]
         public int? ProjectId { get; internal set; }
+
+
+        /// <summary>
+        /// Gets or sets the value indicating whether or not this jobcode should be shared with QuickBooks.
+        /// </summary>
```

Comment:
Please add a <remarks> tag indicating this is a beta-only property, along with instructions to leave the value `null` unless 2-way sync enabled.